.github/workflows/go.yml

```diff
@@ -143,8 +143,6 @@ jobs:
         path: ${{ matrix.test-setups.shortname }}-test-run.log
         retention-days: 5
 
-
-
   # Added to summarize the matrix (otherwise we would need to list every single
   # job in bors.toml)
   tests-result:
```

Comment:
It seems as though the "Go End-to-End Test Results" job could also be renamed in the same pattern (to "e2e results" perhaps)?

Reply:
Makes sense.

@cevian repo owner will need to amend branch protection options to mark new check name as required one.